plugin/path_login.go

```diff
@@ -326,6 +322,11 @@ func (b *GcpAuthBackend) pathIamRenew(req *logical.Request, role *gcpRole) error
 		return fmt.Errorf(clientErrorTemplate, "IAM", err)
 	}
 
+	roleName, ok := req.Auth.Metadata["role"]
```

Comment:
Will this not block the renewals of already issued tokens?

Reply:
Technically, this check is already done beforehand, so it shouldn't. I can just change it to a parameter.